widgets/CTSQLServer-streamingsource.json

```diff
@@ -44,6 +44,43 @@
           "label": "Database name",
           "name": "dbname",
           "description": "SQL Server database name which needs to be tracked. Note: Change Tracking must be enabled on the database for the source to read the chage data"
+        },
+        {
+          "widget-type": "select",
+          "label": "Generate CDC Sequence Number in output schema",
+          "name": "sqn",
+          "description": "Need the CDC Sequence Number in the output schema",
+          "widget-attributes": {
+            "values": [
+              "true",
+              "false"
+            ],
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "CDC Sequence Number",
+          "name": "cdcnnumber",
+          "description": "Sequence Number to start ingesting number"
```

Comment:
these descriptions aren't needed, it should use whatever descriptions are annotated on the plugin config.

Reply:
ok

Reply:
Will address this.